<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScraperMainWindow</class>
 <widget class="QWidget" name="ScraperMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="ScraperBackground" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#ScraperBackground{
background-color: rgb(45, 52, 54);
}

</string>
   </property>
   <widget class="QLabel" name="BackArrowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel:hover{
background-color: rgb(99, 110, 114);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Back-Arrow/left-arrow.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ScrapeGoogleRadioButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Google</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScrapeLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>300</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-color: none;
}

QLineEdit:focus{
border-style: solid;
border-width:1.5px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(46, 170,250,1), stop:1 rgba(41, 57, 152,1));
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ScrapeDropDownMenu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>580</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Any Time</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Past hour</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Past day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Past week</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Past month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Past year</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="SecondBackgroupScraper" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>431</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#SecondBackgroupScraper{
background-color:black;
border-radius:20px
}</string>
    </property>
    <widget class="QLabel" name="ScrapePagesLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>121</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code Light</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Pages</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ScraperSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox:focus{
border-style: solid;
border-width:1.5px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(46, 170,250,1), stop:1 rgba(41, 57, 152,1));
}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ScrapePushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:none;
background-color: rgb(45, 52, 54);
color: white;
}

QPushButton:hover{
border-style: solid;
border-width:2px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255,154,139,1), stop:1 rgba(255,106,136,1));
}
</string>
    </property>
    <property name="text">
     <string>Scrape</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>720</y>
      <width>400</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
border-radius: 10px;
}

QProgressBar::Chunk{
border-radius:10px;
background-color:  qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(46, 170,250,1), stop:1 rgba(41, 57, 152,1));
}

</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="ScrapeSearchEngineLabel">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>100</y>
      <width>299</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code Light</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Search Engine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="KeywordLabel">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>230</y>
      <width>230</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono Light</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>  Keywords</string>
    </property>
   </widget>
   <widget class="QLabel" name="SepLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>240</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>    (Seperated By Comma)</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScrapeTimeRangeLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>253</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code Light</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string> Time Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScraperErrorMessage">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>710</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:none;
background-color: rgb(45, 52, 54);
color: red;
}
</string>
    </property>
    <property name="text">
     <string>Error Please Make Sure All Required Fields Are Completed</string>
    </property>
   </widget>
   <zorder>SecondBackgroupScraper</zorder>
   <zorder>BackArrowLabel</zorder>
   <zorder>ScrapeGoogleRadioButton</zorder>
   <zorder>ScrapeLineEdit</zorder>
   <zorder>ScrapeSearchEngineLabel</zorder>
   <zorder>KeywordLabel</zorder>
   <zorder>SepLabel</zorder>
   <zorder>ScrapeTimeRangeLabel</zorder>
   <zorder>ScrapeDropDownMenu</zorder>
   <zorder>ScrapePushButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>ScraperErrorMessage</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Images/Back-Arrow.qrc"/>
  <include location="../Images/LoadingWheel.qrc"/>
 </resources>
 <connections/>
</ui>
